database.js

```diff
@@ -0,0 +1,51 @@
+let express = require("express");
+let { Pool } = require("pg");
+let env = require("E:/starter2/env.json");
+const argon2 = require('argon2');
+
+let hostname = "localhost";
+let port = 3000;
+let app = express();
+
+app.use(express.json());
+
+let pool = new Pool(env);
+pool.connect().then(() => {
+    console.log("Connected to database");
+});
+
+app.get('/login', async (req, res) => {
+    let { username, password } = {username:'user1', password:'11111'};
+  
+    try {
+      let queryResult = await pool.query('SELECT * FROM userdata WHERE username = $1', [username]);
+      let user = queryResult.rows[0];
+  
+      if (!user) {
+        await pool.query('INSERT INTO userdata (username, password, loggedin) VALUES ($1, $2, $3)', [
+          username,
+          password,
+          true, 
+        ]);
+  
+        res.sendStatus(200);
+      } else {
+        let passwordMatch = (user.password===password);
```

Comment:
We don't need the curly braces, and since this passwordMatch won't change in this scope, we usually set it to const
```
const passwordMatch = user.password === password;
```